Core/Utilities/Time.py

```diff
@@ -236,6 +236,17 @@ def includes( self, myDateTime ):
       return False
     return True
 
+def queryTime(f):
```

Comment:
I would say that timeThis is not the right method since it is just printing out to the std out. This should rather be reimplemented to use the logger instead. The queryTime is mixing in the timing information to the output result. So, on the client side, for example, we can see the exact time of the work of the service net from the protocol overhead. This is quite useful for the timing optimization. I do not think these methods should merged.